src/ompl/base/samplers/informed/src/PathLengthDirectInfSampler.cpp

```diff
@@ -115,21 +115,24 @@ namespace ompl
             }
             else if (InformedSampler::space_->isCompound())
             {
+              std::cout << "PathLengthDirectInfSampler: Compound StateSpace. State Space Type: " << InformedSampler::space_->getType() << std::endl;
                 // Check that it is SE2 or SE3
                 if (InformedSampler::space_->getType() == STATE_SPACE_SE2 ||
-                    InformedSampler::space_->getType() == STATE_SPACE_SE3)
+                    InformedSampler::space_->getType() == STATE_SPACE_SE3 || 
+                    InformedSampler::space_->getType() == 98)
                 {
                     // Variable:
                     // An ease of use upcasted pointer to the space as a compound space
                     const CompoundStateSpace *compoundSpace = InformedSampler::space_->as<CompoundStateSpace>();
 
-                    // Sanity check
-                    if (compoundSpace->getSubspaceCount() != 2u)
-                    {
-                        // Pout
-                        throw Exception("The provided compound StateSpace is SE(2) or SE(3) but does not have exactly "
-                                        "2 subspaces.");
-                    }
+                    // // Sanity check
+                    // if (compoundSpace->getSubspaceCount() != 2u)
+                    // {
+                    //     std::cout << "state space has " << compoundSpace->getSubspaceCount() << " subspaces" << std::endl;
+                    //     // Pout
+                    //     throw Exception("The provided compound StateSpace is SE(2) or SE(3) but does not have exactly "
+                    //                     "2 subspaces.");
+                    // }
```

Comment:
why was the sanity check removed?

Reply:
@dmsj i put this back in and didn't have any issues planning, is there any reason to keep it removed?

Reply:
@yammineramy this is where you're throwing, i'll comment it back out

Reply:
i put an if statement around this if we're in the new state space to skip it